android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -90,18 +90,40 @@ protected void onResume() {
         sessionRecyclerView.setAdapter(sessionsListAdapter);
         sessionRecyclerView.scrollToPosition(SessionsListAdapter.listPosition);
         sessionRecyclerView.setItemAnimator(new DefaultItemAnimator());
-
-        disposable.add(dbSingleton.getSessionsByLocationNameObservable(location)
-                .subscribe(new Consumer<ArrayList<Session>>() {
-                    @Override
-                    public void accept(@NonNull ArrayList<Session> sessions) throws Exception {
-                        mSessions.clear();
-                        mSessions.addAll(sessions);
-                        sessionsListAdapter.notifyDataSetChanged();
-
-                        handleVisibility();
-                    }
-                }));
+        if(searchText!=null){
```

Comment:
The code is not properly indented. Can you please reformat the whole condition you've written using Ctrl+Alt+L.
Also please make changes in Bookmarks fragment as well.